docs/shared/recipes/plugins/create-install-package.md

```diff
@@ -0,0 +1,159 @@
+
+## Testing your create package
+
+Because your create-my-plugin package will install your plugin package at runtime, both packages must be published in order to run them and see the results. To test your packages without making them publicly available, a `local-registry` target should be present on project in your workspace's root.
+
+```jsonc {% fileName="project.json" %}
+{
+  ...
+  "targets": {
+    "local-registry": {
+      "executor": "@nx/js:verdaccio",
+      "options": {
+        "port": 4873,
+        "config": ".verdaccio/config.yml",
+        "storage": "tmp/local-registry/storage"
+      }
+    }
+  }
+}
+```
+
+_(If you don't have such a target, refer to the following [docs page to generate one](/packages/js/generators/setup-verdaccio))_
+
+By running
+
+```shell
+npx nx local-registry
+```
+
+...a local instance of [Verdaccio](https://verdaccio.org/) will be launched at http://localhost:4873 and the NPM, Yarn and PNPM registries will be configured to point to it. This means that you can safely publish, without hitting npm, and test as if you were an end user of your package.
+
+{% callout type="info" title="Registry Cleanup & Reset" %}
+Note, after terminating the terminal window where the `nx local-registry` command is running (e.g. using `CTRL+c` or `CMD+c`) the registry will be stopped, previously installed packages will be cleaned up and the npm/yarn/pnpm registries will be restored to their original state, pointing to the real NPM servers again.
+{% /callout %}
+
+Next, you can **publish** your packages to your new local registry. All of the generated packages have a `publish` target, so you can simply run:
+
+```shell
+npx nx run-many --targets publish --ver 1.0.0 --tag latest
+```
+
+Once the packages are published, you should be able to test the behavior of your "create package" as follows:
+
+```shell
+npx create-my-plugin test-workspace
+```
+
+## Writing and running e2e tests
+
+When setting up the workspace, you should also have gotten a `my-plugin-e2e` package. This package contains the e2e tests for your plugin, and can be run with the following command:
+
+```shell
+npx nx e2e my-plugin-e2e
+```
+
+Have a look at some of the example tests that were generated for you. When running these tests,
+
+- the local registry will be started automatically
+- a new version of the packages will be deployed
+- then your test commands will be run (usually triggering processes that setup the workspace, just like the user would type them into a command line interface)
+- once the test commands have finished, the local registry will be stopped again and cleaned up
+
+## Publishing your create package
+
+Your plugin and create package will both need to be published to NPM to be useable. Publishing your packages is exactly the same as described [previously](#testing-your-create-package), except that won't launch the local registry but rather publish to the real NPM servers.
```

Comment:
```suggestion
Your plugin and create package will both need to be published to NPM to be useable. Publishing your packages is exactly the same as described [previously](#testing-your-create-package), except that you don't run the `local-registry` task so that the `publish` task will publish to the real NPM servers.
```